nbextensions/usability/move_selected_cells/move_selected_cells.yaml

```diff
@@ -0,0 +1,7 @@
+Type: IPython Notebook Extension
+Name: Move selected cells
+Description: Move selected cell*s* using keybaord shortcuts Alt-up and Alt-down
+Link: https://github.com/ipython-contrib/IPython-notebook-extensions/wiki/
```

Comment:
Done. 
Also added a cautionary note to say that such a functionality will probably available shortly directly in Jupyter.